### PR TITLE
Avatar and badge

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (57)
Screenshot (58)
Screenshot (59)
Screenshot (60)

This is my Avatar and Badge component. Please review it!!

### CODE REVIEW

File: Components/Avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="\css\nav.css">
+    <link rel="stylesheet" href="\css\doc.css">
+    <link rel="stylesheet" href="\css\image.css">
+    <link rel="stylesheet" href="\Components\Avatar\avatar.css">
+    <title>Document</title>
```
Comment (dev-enforced) [inline Components/Avatar/avatar.html]: Do name it as Avatar since it is related to avatar component right

File: Components/Badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="\css\nav.css">
+    <link rel="stylesheet" href="\css\doc.css">
+    <link rel="stylesheet" href="\Components\Badge\badge.css">
+    <link rel="stylesheet" href="\Components\Avatar\avatar.css">
+    <title>Document</title>
```
Comment (dev-enforced) [inline Components/Badge/badge.html]: Do name it to Badge

**Review by dev-enforced — APPROVED** [review]
The code looks good to me. Suggested some minor changes. The CSS files are also well written common classes and refactoring done good here.

File: Components/Avatar/avatar.html (hunk 1 above)
Comment (Megha-Joshi) [reply Components/Avatar/avatar.html]: Okay! I'll change it

**Review by Omkar-Ghate — APPROVED** [review]
Great work. Good to merge.

**Pull request merged**